src/styles/utils/_variables.scss

```
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$font-size: 30px;
+$margin: 1px;
+$border: 1px solid #000;
+
+$start-day:
+  "monday" 0,
+  "tuesday" calc(#{$cell-size} + #{$margin}),
+  "wednesday" calc(2 * (#{$cell-size} + #{$margin})),
+  "thurthday" calc(3 * (#{$cell-size} + #{$margin})),
```

You can calc it just one time instead calc each time
Also, you don't need `calc` in SCSS for calculations
```suggestion
$border: 1px solid #000;
$day-margin: $cell-size + $margin;
$start-day:
  "monday" 0,
  "tuesday" $day-margin,
  "wednesday" 2 * $day-margin,
  "thurthday" 3 * $day-margin,
```